lax/lichens/sciencerun2.py

```diff
@@ -229,7 +229,49 @@ def _process(self, df):
 
 # S1 single scatter
 # Contact: Joran
-S1SingleScatter = sr1.S1SingleScatter
+class S1SingleScatter(Lichen):
+    """Requires only one valid interaction between the largest S2, and any S1 recorded before it.
+
+    The S1 cut checks that any possible secondary S1s recorded in a waveform, could not have also
+    produced a valid interaction with the primary S2. To check whether an interaction between the
+    second largest S1 and the largest S2 is valid, we use the S2Width cut. If the event would pass
+    the S2Width cut, a valid second interaction exists, and we may have mis-identified which S1 to
+    pair with the primary S2. Therefore we cut this event. If it fails the S2Width cut the event is
+    not removed.
+    
+    Requires Extended minitrees. 
+
+    The cut is investigated for SR2 here:
+    https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenon1t:analysis:sciencerun2:s1singlescattercut
+
+    It should be applicable to data regardless whether it is ER or NR.
+    Contacts: Jacques Pienaar, <jpienaar@uchicago.edu>
+    (for SR2) Joran Angevaare, <j.angevaare@nikhef.nl>  
+    """
+    
+    version = 5
+    s2width = sciencerun1.S2Width
```

Comment:
Can you let this point to the SR2 S2width cut? At the moment this in turn just points to the SR1 S2 Width, but that may change.